cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/config_utils.py

```diff
@@ -1,6 +1,15 @@
+import inspect
 import math
+from pathlib import Path
 from typing import List, Tuple
 
+
+def get_template_folder() -> str:
```

Comment:
Maybe let's call this function something like `get_slurm_config_template_folder()`?

Reply:
(See comment above) I kept it general since the function can easily be generalized (by simply making the function name a parameter) and used in a more general context.
I didn't do this now, since we do not yet need it.